programs/ziti-edge-tunnel/include/model/dtos.h

```diff
@@ -82,6 +86,7 @@ XX(Id, string, none, Id, __VA_ARGS__) \
 XX(Name, string, none, Name, __VA_ARGS__) \
 XX(Protocols, string, array, Protocols, __VA_ARGS__) \
 XX(Addresses, tunnel_address, array, Addresses, __VA_ARGS__) \
+XX(AllowedSourceAddresses, allowed_source_address, array, AllowedSourceAddresses, __VA_ARGS__) \
```

Comment:
make it a `list` -- it is easier to deal with

Reply:
why not use `ziti_address` instead of a new type?

Reply:
Not clear to me how I would do that.  Seems all the items in the tunnel events are newly defined types in model/dtos.h which in some cases are populated with information from ziti_address i.e tunnel_address.  However they do not use it as a type directly.  Maybe I'm not interpreting your suggestion correctly.

Reply:
Also seems all the other multi item entries in model/dtos.h associated with a TUNNEL_SERVICE are defined as arrays i.e.
XX(Protocols, string, array, Protocols, __VA_ARGS__) 
XX(Addresses, tunnel_address, array, Addresses, __VA_ARGS__) 
XX(Ports, tunnel_port_range, array, Ports, __VA_ARGS__)